app/src/views/search-engine/components/search-bar/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { AppBar, Toolbar, Avatar, Input } from '@material-ui/core';
+import styles from './styles';
+
+const SearchBar = ({ setSearch }) => {
+  const classes = styles();
+
+  return (
+    <AppBar position="sticky" color="inherit" className={classes.appbar}>
+      <Toolbar>
+        <div className={classes.search}>
+          <Input
+            type="search"
+            placeholder="Buscar..."
+            classes={{
+              root: classes.inputRoot,
+              input: classes.inputInput,
+            }}
+            onChange={({ target: { value } }) => setSearch(value)}
+          />
+        </div>
+        <Avatar alt="Reglamento Vial" src="/static/icon-bg.png" className={classes.avatar} />
```

Comment:
Maybe this image could be the trigger button as well, it already seems to be in the right position anyways and it currently serves no other purpose.

Reply:
Yeah, I'll go ahead and upgrade that image to a trigger. It doesn't hurt anyone.

Reply:
I'll be adding an additional search icon adjacent to the search bar.